### PR TITLE
*: improve log

### PR DESCRIPTION
**Pull request by coocood**
Add logging for set system variable.
Add connection ID in log.
Adjust some log level.
Improve readability.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

**Comment by hanfei1991** [comment]
LGTM

File: server/conn.go [hunk 1]
```diff
@@ -593,9 +593,9 @@ func (cc *clientConn) handleQuery(sql string) (err error) {
 		sql = sql[:1024]
 	}
 	if costTime < time.Second {
-		log.Debugf("[TIME_QUERY] %v %s", costTime, sql)
+		log.Debugf("[TIME_QUERY][%d] %v, session:%d\n%s", cc.connectionID, costTime, sql)
```
Comment (zimulala) [inline server/conn.go]: The number of arguments doesn't match.
The same as line 598.

**Comment by shenli** [comment]
LGTM

File: server/server.go [hunk 2]
```diff
@@ -113,6 +112,7 @@ func (s *Server) newConn(conn net.Conn) *clientConn {
 		collation:    mysql.DefaultCollationID,
 		alloc:        arena.NewAllocator(32 * 1024),
 	}
+	log.Infof("[%d] newConn %s", cc.connectionID, conn.RemoteAddr().String())
```
Comment (ngaut) [inline server/server.go]: s/newConn/new connection/

File: server/server.go [hunk 3]
```diff
@@ -194,7 +194,7 @@ func (s *Server) onConn(c net.Conn) {
 		return
 	}
 	defer func() {
-		log.Infof("close %s", conn)
+		log.Infof("[%d] close conn", conn.connectionID)
```
Comment (ngaut) [inline server/server.go]: s/conn/connection/

File: tidb_test.go [hunk 4]
```diff
@@ -311,8 +312,12 @@ func newStore(c *C, dbPath string) kv.Storage {
...
 }
 
+var testConnID uint64
+
 func newSession(c *C, store kv.Storage, dbName string) Session {
 	se, err := CreateSession(store)
+	id := atomic.AddUint64(&testConnID, 1)
+	se.SetConnectionID(id)
 	c.Assert(err, IsNil)
 	se.Auth("root@%", nil, []byte("012345678901234567890"))
```
Comment (ngaut) [inline tidb_test.go]: Please using ` instead of "

**Comment by coocood** [comment]
@ngaut PTAL

**Comment by ngaut** [comment]
LGTM

**Pull request merged**